pkg/nodeinfo/attributes.go

```diff
@@ -43,22 +43,25 @@ type AttributeType int
 
 // Attribute type Enum, add new types before Last and update the mapping below
 const (
-	AttrTypeOS = iota
-	AttrTypeKernel
+	AttrTypeOSNameFull = iota
 	AttrTypeHostname
 	AttrTypeCPUArch
+	AttrTypeOSName
```

Comment:
Why do you need OSNameFull? I understand it just OSName + OSVer

Reply:
originally i started removing it (and eventually reverted it), it introduced quite abit of code and template changes that hinders readability of this PR.

technically you dont need, it can be handled with the two newly added attributes, its currently kept for convenience.
there can be a separate PR to remove it.

Reply:
ok lets open github issue and assign @Mmduh-483